gulp/tasks/createPackage.js

```diff
@@ -55,13 +55,14 @@ gulp.task('select-view', (cb) => {
 })
 
 gulp.task('create-package', ['select-view', 'custom-js','custom-scss','custom-css'], function () {
+    const env = process.env.NODE_ENV;
     const code = config.view();
     console.log('Creating package for : ('+code+'.zip)');
     console.log(code);
     console.log(' in  : /packages');
     console.log('\r\n');
     console.log('............................................................................................................................................');
     return gulp.src(['./primo-explore/custom/'+code,'./primo-explore/custom/'+code+'/html/**','./primo-explore/custom/'+code+'/img/**','./primo-explore/custom/'+code+'/css/custom1.css','./primo-explore/custom/'+code+'/js/custom.js'], {base: './primo-explore/custom'})
-        .pipe(zip(code+'.zip'))
+        .pipe(zip(`${code}.${new Date().toISOString().replace(/[^0-9]/g,'').slice(0, 12)}.${env === 'production' || !env ? 'production' : env}.zip`))
```

Comment:
this is a very welcome change. 👍

Reply:
@atla5 thanks! It works great with our CI/CD. much less confusing